frontend/src/components/card-content/check-route/select-point.tsx

```
@@ -1,46 +1,63 @@
-	point: RoutePoint | null, isPointSelecting: boolean, setPoint: Function, setIsPointSelecting: Function
+export default function SelectPoint({
+  isStarting,
+  point,
+  setPoint,
+  isPointSelecting,
+  setIsPointSelecting,
+}: {
+  isStarting: boolean;
+  point: RoutePoint | null;
+  isPointSelecting: boolean;
+  setPoint: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)